src/frontend/src/btc/components/send/BtcSendTokenWizard.svelte

```diff
@@ -44,10 +59,44 @@
 
 	// TODO: implement send function when related services are ready
 	const send = () => {};
+
+	const selectUtxosFee = async () => {
```

Comment:
is this use anywhere else than in the review step?
if no, shouldn't we move this within that particular component?

Reply:
actually, why not having this logic directly in `BtcUtxosFee`?

Reply:
Good point. We can only keep the value in the wizard (it will be needed for the send step), but the fetching logic can be moved to the child component. It's now done like this.